jsk_2016_01_baxter_apc/euslisp/main.l

```diff
@@ -213,6 +213,8 @@
           (send *ri* :wait-interpolation)
           (send *ri* :stop-grasp arm) ;; release object
           (send *ri* :spin-off-by-wrist arm :times 20)
+          (send *ri* :gripper-servo-on arm)
```

Comment:
In `return_object`, `move-arm-body->order-bin` doesn't exist, so this isn't needed.